spec/controllers/reviews_controller_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'spec_helper'
+
+describe ReviewsController do
+
+  describe 'POST :create' do
+
+    let(:video) { video = Fabricate(:video) }
+
+    context 'user is authorized' do
+      let(:user) { user = Fabricate(:user) }
+      before do
+        self.controller.login_user user
```

Comment:
I built a method ```sign_in user``` by creating a support/AuthHelper module which includes the module from ApplicationController::AuthMethods.  Have a look at: https://github.com/jepetko/myflix/commit/2132d69b4563cb27b3e5d1ed28716f9690c0be9e.